### PR TITLE
Feature: Added support for extracting multiple archives at same time.

### LINKED ISSUE #9949: Feature: Add support for extracting multiple archives at the same time
**Requirements**
- Display extract button on toolbar and in the context menu when multiple archives are selected
   - Add sub item to `Extract here`
   - Add sub item to `Extract to "*\"`

### PR DESCRIPTION
**Pull request by puppetsw**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #9949

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
image

### CODE REVIEW

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml [hunk 1]
```diff
@@ -1,930 +1,933 @@
...
+                    LabelPosition="Default"
+                    Style="{StaticResource ToolBarAppBarButtonFlyoutStyle}">
+                    <AppBarButton.Content>
+                        <local:ColoredIcon BaseLayerGlyph="&#xF03F;" OverlayLayerGlyph="&#xF040;" />
+                    </AppBarButton.Content>
+                    <AppBarButton.Flyout>
+                        <MenuFlyout helpers:MenuFlyoutHelper.IsVisible="{x:Bind ViewModel.CanExtract, Mode=OneWay}" Placement="Bottom">
+                            <MenuFlyoutItem
+                                x:Name="ExtractSingle"
+                                x:Load="{x:Bind ViewModel.IsMultipleArchivesSelected, Mode=OneWay, Converter={StaticResource BoolNegationConverter}}"
```
Comment (yaichenbaum) [inline src/Files.App/UserControls/InnerNavigationToolbar.xaml:227]: Looks like this xaml file was swittched to spaces, can you set it back to tabs?

File: src/Files.App/Interacts/BaseLayoutCommandImplementationModel.cs [hunk 2]
```diff
@@ -623,151 +625,105 @@ public async Task DecompressArchive()
...
-                        destinationFolder = await FilesystemTasks.Wrap(() => parentFolder.CreateFolderAsync(Path.GetFileName(destinationFolderPath), CreationCollisionOption.GenerateUniqueName).AsTask());
-                    }
-                    if (destinationFolder == null)
-                    {
-                        return; // Could not create dest folder
-                    }
+            if (option != ContentDialogResult.Primary)
+            {
+                return;
+            }
```
Comment (yaichenbaum) [inline src/Files.App/Interacts/BaseLayoutCommandImplementationModel.cs:642]:
```suggestion
            if (option != ContentDialogResult.Primary)
                return;
```

Brackets are only used on multiline if statements

Comment (puppetsw) [reply src/Files.App/Interacts/BaseLayoutCommandImplementationModel.cs]: I could've sworn the style guide said brackets for all. Must've of been a different one. Updated.

Comment (yaichenbaum) [reply src/Files.App/Interacts/BaseLayoutCommandImplementationModel.cs]: Style guide was recently updated 🙂

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Pull request merged**